platform/local_country_file_utils.hpp

```diff
@@ -18,7 +18,8 @@ namespace migrate
 }
 
 // Removes all files downloader creates during downloading of a country.
-void DeleteDownloaderFilesForCountry(CountryFile const & countryFile, int64_t version);
+void DeleteDownloaderFilesForCountry(CountryFile const & countryFile,
```

Comment:
Please, replace folder by dataDir too. Also, create a versions that do not take a string parameter and just calls these new functions with Platform::WritableDir(), from my point of view it's quite obfuscating that empty dataDir means Platform::WritableDir().